run.sh

```diff
@@ -28,6 +29,8 @@ core config:
   dbpass: $DB_PASS
   dbname: $DB_NAME
   dbhost: db:3306
+  extra-php: |
+    define('WP_DEBUG', ${WP_DEBUG,,});
```

Comment:
It returns the lowercase version of `WP_DEBUG`. For some reason docker-compose would capitablize the environment variable.